Source/WebKit/Shared/Extensions/WebExtensionContentWorldType.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
```

Comment:
Should this copyright date be 2023?

Reply:
I'll get it right one of these years. Maybe next year…